.prettierrc.js

```diff
@@ -0,0 +1,6 @@
+module.exports = {
+  trailingComma: "es5",
+  tabWidth: 2,
+  semi: false,
```

Comment:
Let's keep the semicolons.
```suggestion
  semi: true,
```